File: src/main/java/lib/model/Address.java
package lib.model;

/**
 * Created by gzae on 12/13/16.
 */
public class Address {

    public int adressID;
    public String name;
    public String street;
    public String streetnr;
    public String zip;
    public String city;
    public String country;

    public Address(int ID, String _name, String _street, String _streetnr, String _zip, String _city, String _country)
    {
        this.adressID = ID;
        this.name = _name;
        this.street = _street;
        this.streetnr = _streetnr;
        this.zip = _zip;
        this.city = _city;
        this.country = _country;
    }
}

File: src/main/web/src/app/shared/model/address.ts
import {BaseObject} from "./util/base-object";

export interface Address extends BaseObject {
	name: string;
	street: string;
	streetNr: string;
	zip: string;
	city: string;
	country: string;
	user: number;
	item: number;
	latitude: number;
	longitude: number;

}

export function createAddress(): Address {
	return {
		id: -1,
		name: "",
		street: "",
		streetNr: "",
		zip: "",
		city: "",
		country: "",
		user: null,
		item: null,
		latitude: 0,
		longitude: 0,
	};
}
